### DISCUSSION TITLE
How to translate the coordinates of a polygon?

### DISCUSSION THREAD

**Discussion by szlq** [NetTopologySuite/NetTopologySuite#717] [Q&A] [answered]
How to translate the coordinates of a polygon. For example, increasing the X coordinates of a polygon by 10 meters.

**Comment by szlq** [comment, marked as answer]
use ICoordinateSequenceFilter and Geometry.Apply()

**Reply by FObermaier** [reply]
Implementing `IEntireCoordinateSequenceFilter` would be the better choice.
